### PR TITLE
reportbug: add page

### PR DESCRIPTION
**Pull request by Rudloff**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/bugreport.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# bugreport
+
+> File a bug report on Debian.
```
Comment (andrik) [inline pages/linux/bugreport.md:3]:
```suggestion
> Bug report tool of Debian distribution.
```

File: pages/linux/bugreport.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`reportbug {{package}}`
+
+- File a report about a package that does not exist yet (to request it, for example):
+
+`reportbug wnpp`
+
+- Report a bug that is not about a specific package (general problem, infrastructure, etc.):
+
+`reportbug other`
```
Comment (andrik) [inline pages/linux/bugreport.md:16]: I read the documentation and it looks like to me that these two examples are referring to pseudo-package, right? Could we merge them?

Comment (Rudloff) [reply pages/linux/bugreport.md:16]: `reportbug other` will list all the pseudopackages and you can select `wnpp` from there so I guess it is a bit redundant.
Do you think we can just drop the `wnpp` example?

Comment (andrik) [reply pages/linux/bugreport.md:16]: Yeah, in that case, I think we can drop it.

**Pull request merged**